core/domain/voiceover_services.py

```diff
@@ -59,21 +59,27 @@ def _get_entity_voiceovers_from_model(
         EntityVoiceovers. An instance of EntityVoiceovers object, created from
         its model.
     """
-    content_id_to_voiceovers_dict = {}
+    content_id_to_voiceovers: Dict[str, Dict[
+        feconf.VoiceoverType, Optional[state_domain.Voiceover]]] = {}
     for content_id, voiceover_type_to_voiceover_dict in (
             entity_voiceovers_model.voiceovers.items()):
-        content_id_to_voiceovers_dict[content_id] = {
```

Comment:
Ditto, we need to update the implementation because now we are allowing None values in place of automatic voiceovers.